src/components/WeeklySummariesReport/WeeklySummariesReport.test.jsx

```diff
@@ -40,9 +40,9 @@ describe('WeeklySummariesReport page', () => {
       render(<WeeklySummariesReport {...props} />);
     });
 
-    it('should have 3 tab', () => {
+    it('should have 4 tab', () => {
       const li = screen.getAllByRole('listitem');
-      expect(li.length).toEqual(3);
+      expect(li.length).toEqual(4);
```

Comment:
We can add a _jest.clearAllMocks()_ call in the _afterEach_ block to clear all the mocks created in each test case:

afterEach(() => {
      jest.clearAllMocks();
    });